styles/styles.css

```diff
@@ -0,0 +1,270 @@
+* {
+  box-sizing: border-box;
+}
+button, input, select {
+  font-family: inherit;
+  font-size: 100%;
+}
+header {
+  background-image: url(../images/header-image.jpg);
+  background-size: cover;
+}
+footer {
+  color: white;
+  background-color: rgb(13, 45, 78);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 3rem 0 3rem;
+}
+body {
+  margin: auto;
+  font-size: 16px;
+}
+main {
+  max-width: 1440px;
+}
+.logo {
+  padding: 3rem 0rem 0rem 6rem;
+  width: 100%;
+  min-width: 320px;
+}
+.tagline {
+  font-size: 1.55rem;
+  margin-top: -1rem;
+  padding: 0rem 0rem 10rem 6rem;
+  color: rgb(116, 171, 42);
+}
+.cost {
+  color: white;
+}
+.checkout {
+  color: white;
+  font-weight: bold;
+  background-color: rgb(116, 171, 42);
+  padding: .01rem 2rem;
+  border-radius: .1rem;
+}
+.checkout-link {
+  text-decoration: none;
+}
+.cart {
+  width: 15%;
+  padding-right: .5rem;
+}
+.cart-cost {
+  display: flex;
+  justify-content: flex-end;
+}
+.cart-cost-button {
+  padding: 3rem 6rem 0rem 0rem;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+.top-nav {
+  background-color: rgb(231, 231, 231);
+  width: 100%;
+  padding: .5rem 0rem .5rem 12rem;
+}
+.top-nav a {
+  text-decoration: none;
+  color: rgb(11, 38, 66);
+}
+li {
+  display: inline;
+  text-decoration: none;
+  font-weight: bold;
+}
+li::after {
+  content: "/";
+  padding: .8rem;
+}
+li:last-of-type:after {
```

Comment:
there is a way to shorthand these afters. look into using `not` . I totally used the same code as you, but saw a few students using `not` and it was great!